### PR TITLE
upgrade tinacms to prerelease

### PR DESCRIPTION
**Pull request by ncphillips**

### CODE REVIEW

**Comment by dwalkr** [comment]
will this fix https://github.com/tinacms/tinacms.org/issues/669 ?

**Comment by ncphillips** [comment]
I think so yes

File: package.json [hunk 1]
```diff
@@ -54,7 +54,7 @@
     "next": "^9.5.0",
     "next-seo": "^4.1.0",
     "next-svgr": "0.0.2",
-    "next-tinacms-github": "^0.27.0",
+    "next-tinacms-github": "^0.25.0-alpha.0",
```
Comment (DirtyF) [inline package.json:57]: https://github.com/tinacms/tinacms/blob/master/packages/next-tinacms-github/package.json#L3

Comment (ncphillips) [reply package.json:57]: Bad `yarn add` thanks

**Pull request merged**